code_schemes/rqas/aik/aik_communities.json

```diff
@@ -13,6 +13,182 @@
             "VisibleInCoda": true,
             "Shortcut": "e"
         },
+        {
```

Comment:
Would it help to have this an alphabetical order given the length?

Reply:
yes. Updated, thanks.